runtime/expr/function/compare.go

```diff
@@ -0,0 +1,28 @@
+package function
+
+import (
+	"github.com/brimdata/zed"
+	"github.com/brimdata/zed/runtime/expr"
+)
+
+func NewCompare() *Compare {
+	return &Compare{
+		fn: expr.NewValueCompareFn(true),
+	}
+}
+
+type Compare struct {
+	fn expr.CompareFn
```

Comment:
We usually call `cmp` or `compare`.
```suggestion
	cmp expr.CompareFn
```